README.md

```diff
@@ -2,15 +2,15 @@ Docker Compose
 ==============
 ![Docker Compose](logo.png?raw=true "Docker Compose Logo")
 
-*(Previously known as Fig)*
+Compose is a tool for defining and running multi-container Docker applications.
```

Comment:
I'm worried that "With compose, you create a `docker-compose.yml`" makes it sound like compose is creating the config file for you.  

It's technically "service configuration" not just container configuration.

How does this work for the middle sentence: 

... Docker applications. WIth Compose, you use a Compose file to configure your application's services.  Then, using a single command...